cluster-operator/src/main/java/io/strimzi/operator/cluster/model/VolumeUtils.java

```diff
@@ -36,8 +43,28 @@
  * Shared methods for working with Volume
  */
 public class VolumeUtils {
-
     protected static final Logger log = LogManager.getLogger(VolumeUtils.class.getName());
+    private static Pattern volumeNamePattern = Pattern.compile("^([a-z0-9]{1}[a-z0-9-]{0,61}[a-z0-9]{1})$");
+
+    /**
+     * Creates a Kubernetes volume which will map to ConfigMap
+     *
+     * @param name              Name of the Volume
+     * @param configMapName     Name of the ConfigMap
+     * @return                  The newly created Volume
+     */
+    public static Volume createConfigMapVolume(String name, String configMapName) {
+        ConfigMapVolumeSource configMapVolumeSource = new ConfigMapVolumeSourceBuilder()
+                .withName(configMapName)
+                .build();
+
+        Volume volume = new VolumeBuilder()
+                .withName(getValidVolumeName(name))
+                .withConfigMap(configMapVolumeSource)
+                .build();
+        AbstractModel.log.trace("Created configMap Volume named '{}' with source configMap '{}'", name, configMapName);
```

Comment:
I am curious, why use the AbstractModel log object as opposed to one inside this class?

Reply:
Thanks for noticing. That was just refactoring issue. Should be fixed now.